continuous_delivery/models/build_configuration.py

```diff
@@ -13,9 +13,17 @@ class BuildConfiguration(Model):
         'type': {'key': 'type', 'type': 'str'},
         'working_directory': {'key': 'workingDirectory', 'type': 'str'},
         'node_type': {'key': 'NodeJsTaskRunner', 'type': 'str'},
+        'python_framework': {'key': 'pythonFramework', 'type': 'str'},
+        'python_version': {'key': 'pythonExtensionId', 'type': 'str'},
+        'django_settings_module': {'key': 'djangoSettingsModule', 'type': 'str'},
+        'flask_project_name': {'key': 'flaskProjectName', 'type': 'str'}
```

Comment:
As discussed offline, we are going ahead with common class for all app-types i.e. NodeJsBuildConfiguration. Hence, no change is required!